cogs/induct.py

```diff
@@ -181,6 +181,11 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             name="introductions",
         )
 
+        initial_response: discord.Interaction | discord.WebhookMessage = await ctx.respond(
+                ":hourglass: Processing Induction... :hourglass:",
```

Comment:
Invalid indentation. Function arguments should be indented by one level, closing bracket should **not** be indented.